src/main/java/de/digitalcollections/iiif/hymir/image/business/ImageServiceImpl.java

```diff
@@ -343,6 +343,9 @@ private DecodedImage readImage(
     } finally {
       metrics.clearTimer(metricKey);
       if (reader != null) {
+        if (reader.getInput() instanceof ImageInputStream) {
+          ((ImageInputStream) reader.getInput()).close();
+        }
```

Comment:
Should we do the same in `readImageInfo`?  https://github.com/dbmdz/iiif-server-hymir/blob/80321ee8a72cd0d65263d3edfbc2b70ebc6724ea/src/main/java/de/digitalcollections/iiif/hymir/image/business/ImageServiceImpl.java#L224-L229

Reply:
Absolutely, thank you!